code/.gitignore

```diff
@@ -119,6 +119,13 @@ secrets.js
 
 webclient/dist
 
+# DS_Store files
+**/.DS_Store
+
+# Build Files
+build/*
```

Comment:
just contracts, not abi. Merge master?

Reply:
Have merged with master and also updated to 
```
# Build Files
build/contracts
```
🙏